controllers/argocd/statefulset.go

```diff
@@ -505,3 +511,15 @@ func (r *ReconcileArgoCD) triggerStatefulSetRollout(sts *appsv1.StatefulSet, key
 	sts.Spec.Template.ObjectMeta.Labels[key] = nowNano()
 	return r.Client.Update(context.TODO(), sts)
 }
+
+//to update nodeSelector and tolerations in reconciler
+func updateNodePlacementStateful(existing *appsv1.StatefulSet, ss *appsv1.StatefulSet, changed *bool) {
```

Comment:
Please add a test to verify the behavior of this function

Reply:
sure